src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -339,7 +339,7 @@ std::optional<std::string> hashStringSHA1(std::string& input)
     constexpr int OS_SHA1_HEXDIGEST_SIZE = (SHA_DIGEST_LENGTH * 2);
     constexpr int OS_SHA1_ARRAY_SIZE_LEN = OS_SHA1_HEXDIGEST_SIZE + 1;
 
-    char* parameter = NULL;
+    char* parameter = nullptr;
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#modern-language-features-and-best-practices

Use C++ nullptr instead of NULL.